src/SolidColor.elm

```diff
@@ -1,27 +1,33 @@
-module Color exposing
-    ( Color
-    , fromRGB, fromHSL, fromHex
-    , toRGBString, toHSLString
+module SolidColor exposing
```

Comment:
how would you feel about `Color.Solid` as a name, exposing `Color`? I think:

- it's clear that it's a specialization of `Color`, in the way that `Array.Hamt` was a specialization of `Array`
- it's a little nicer in the type signatures. `import Color.Solid as Solid` lets you do `Solid.Color`.

Reply:
Hmm yeah! I could see that